### ISSUE TITLE
moving handles and logs, help

### ISSUE THREAD

**Issue by fishasses** [illuminatedwax/pesterchum#182]
is there any way to move all my chumtags and logs from those chumtags into pesterchum on a new computer through a pen drive or other option? I don't want to lose my logs from my old handles and I'm selling my old laptop soon, so I need help..

**Comment by slumberingsorceress** [comment]
Yup just log into pesterchum, go to the third item in the drop down menu and once there select the log directory. You have access to all your logs from your handles from there. 

As for transferring chumtags, I think you need to access the profiles folder. If I remember correctly you can just copy those files into the same file location on your new computer.
